website/docs/r/policy_lb_server_ssl_profile.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+subcategory: "Load Balancer"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_lb_server_ssl_profile"
+description: A resource to configure a Load Balancer Server SSL Profile.
+---
+
+# nsxt_policy_lb_server_ssl_profile
+
+This resource provides a method for the management of a Load Balancer Server SSL Profile.
+
+This resource is applicable to NSX Policy Manager.
+
+## Example Usage
+
+```hcl
+resource "nsxt_policy_lb_server_ssl_profile" "test" {
+  display_name          = "test"
+  description           = "Terraform provisioned profile"
+  cipher_group_label    = "BALANCED"
+  session_cache_enabled = true
+}
+```
+
+## Custom Cipher Profile Example Usage
+
+```hcl
+resource "nsxt_policy_lb_server_ssl_profile" "test" {
+  display_name          = "test"
+  description           = "Terraform provisioned profile"
+  cipher_group_label    = "CUSTOM"
+  ciphers               = ["TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384", "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384"]
+  session_cache_enabled = false
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `display_name` - (Required) Display name of the resource.
+* `description` - (Optional) Description of the resource.
+* `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
+* `nsx_id` - (Optional) The NSX ID of this resource. If set, this ID will be used to create the resource.
+* `cipher_group_label` - (Required) One of `BALANCED`, `HIGH_SECURITY`, `HIGH_COMPATIBILITY` or `CUSTOM`.
+* `ciphers` - (Optional) Supported SSL cipher list. Can only be specified if `cipher_group_label` is set to `CUSTOM`, otherwise the list is assigned by NSX.
+* `protocols` - (Optional) In newer versions of NSX, only TLS1.2 is supported.
```

Comment:
This was taken from the OpenAPI spec, but maybe for Terraform docs users would like to know what "newer" means. I can find that out.

Reply:
I actually wrote that one :) Depending on how old ago non-TLS1.2 was removed from NSX, we might want to consider removing this attribute altogether.

Reply:
The "newer versions of NSX" start with 4.2